tests/onnx/test_nncf_graph_builder.py

```diff
@@ -0,0 +1,32 @@
+import os
+import pytest
+
+from nncf.experimental.onnx.graph.nncf_graph_builder import GraphConverter
+
+from tests.onnx.models import LinearModel
+from tests.onnx.models import MultiInputOutputModel
+
+import networkx as nx
+
+TEST_MODELS = [LinearModel(), MultiInputOutputModel()]
+
+
+@pytest.mark.parametrize("model", TEST_MODELS)
+def test_built_nncf_graphs(model):
```

Comment:
looks similar to `check_nx_graph`. maybe alignment would be better than a new function with some duplication of code

Reply:
I prefer to not import functions from other frameworks (TF or PT)

Reply:
you can create common utils, if similar functionality is used cross frameworks

Reply:
They are not the same

Reply:
Created the ticket - 76508